webui/src/pages/index.jsx

```diff
@@ -39,78 +39,81 @@ import PolicyPage from "./auth/policies/policy";
-              <Route path="tags" element={<RepositoryTagsPage />} />
-              <Route path="compare/*" element={<RepositoryComparePage />} />
-              <Route path="actions">
-                <Route index element={<RepositoryActionsPage />} />
-                <Route path=":runId" element={<RepositoryActionPage />} />
-              </Route>
-              <Route path="settings" element={<SettingsLayout />}>
-                <Route index element={<Navigate to="general" />} />
-                <Route path="general" element={<RepositoryGeneralSettingsPage />} />
-                <Route path="retention" element={<RepositoryRetentionPage />} />
-                <Route path="branches" element={<RepositorySettingsBranchesPage />} />
-              </Route>
-              <Route index element={<Navigate to="objects" />} />
-            </Route>
-          </Route>
-          <Route path="auth" element={<Layout logged={false} />}>
-            <Route index element={<Navigate to="credentials" />} />
-            <Route path="login" element={<LoginPage />} />
-            <Route path="users/create" element={<ActivateInvitedUserPage />} />
-            <Route element={<AuthLayout />}>
-              <Route path="credentials" element={<CredentialsPage />} />
-              <Route path="users" element={<UsersIndexPage />}>
-                <Route index element={<UsersPage />} />
-                <Route path=":userId">
-                  <Route index element={<Navigate to="groups" />} />
-                  <Route path="groups" element={<UserGroupsPage />} />
-                  <Route exact path="policies" element={<UserPoliciesPage />} />
-                  <Route exact path="policies/effective" element={<UserEffectivePoliciesPage />} />
-                  <Route exact path="credentials" element={<UserCredentialsPage />} />
-                </Route>
-              </Route>
-              <Route path="groups">
-                <Route index element={<GroupsPage />} />
-                <Route path=":groupId">
-                  <Route index element={<Navigate to="members" />} />
-                  <Route path="members" element={<GroupMembersPage />} />
-                  <Route path="policies" element={<GroupPoliciesPage />} />
-                </Route>
-              </Route>
-              <Route path="policies">
-                <Route index element={<PoliciesPage />} />
-                <Route path=":policyId" element={<PolicyPage />} />
-              </Route>
-            </Route>
-          </Route>
-          <Route path="/setup" element={<Layout logged={false} />}>
-            <Route index element={<Setup />} />
-            <Route path="*" element={<Setup />} />
-          </Route>
-          <Route path="*" element={<Navigate to="/repositories" replace />} />
-        </Routes>
-      </WithLoginConfigContext>
-    </Router>
-  );
+    return (
+        <Router>
+            <WithAppContext>
```

Comment:
Besides lint changes, the only change in this file is adding wrapping with `WithAppContext`.